magento/pwa-studio
# Background repeat not working on safari

## Description

When creating a banner / column / row / tabItem with a background, the background-repeat property is automatically set to true on safari

1. Change node.style.backgroundRepeat !== 'no-repeat' test in utils.js to just node.style.backgroundRepeat
2. Then change backgroundRepeat ? 'repeat' : 'no-repeat'; to backgroundRepeat; in banner.js, column.js and row.js
3. Change propTypes and propTypes comments value in the same files

## Related Issue







Closes #3347.

## Verification Steps

1. Create a CMS page
2. Insert a banner with an image with low width
3. Set the Background Repeat property to no
4. Save page
5. See the result in front on Safari, check if the background is not repeated

#### Is Browser/Device testing needed?

-   Device : iPhone8, Mac and iPad Pro]:
-   Browser : Safari
-   Browser Version : Version 14.0.3 (16610.4.3.1.7)
